src/status_im/components/toolbar_new/view.cljs

```diff
@@ -13,10 +13,17 @@
             [status-im.components.context-menu :refer [context-menu]]
             [status-im.components.toolbar-new.actions :as act]
             [status-im.components.toolbar-new.styles :as st]
-            [status-im.accessibility-ids :as id]
             [status-im.utils.platform :refer [platform-specific]]
             [reagent.core :as r]))
 
+(defn create-nav-button [nav-action]
```

Comment:
we can't desctruct `nav-image` as `image` in parameters because this symbol will shadow `status-im.components.react/image` in current scope. If we refactor this ns and use alias for `status-im.components.react` we can use `image` in parameters destructuring form, then no need for `nav-image`